code/__DEFINES/maps.dm

```diff
@@ -27,5 +27,8 @@ Last space-z level = empty
 #define EMPTY_AREA_7 "Empty Area 7"
 #define EMPTY_AREA_8 "Empty Area 8"
 #define AWAY_MISSION "Away Mission"
-#define AWAY_MISSION_LIST list(AWAY_MISSION = SELFLOOPING)
-#define DEFAULT_MAP_TRANSITION_CONFIG list(MAIN_STATION = CROSSLINKED, CENTCOMM = SELFLOOPING, EMPTY_AREA_1 = CROSSLINKED, EMPTY_AREA_2 = CROSSLINKED, MINING = SELFLOOPING, EMPTY_AREA_3 = CROSSLINKED, EMPTY_AREA_4 = CROSSLINKED, EMPTY_AREA_5 = CROSSLINKED, EMPTY_AREA_6 = CROSSLINKED, EMPTY_AREA_7 = CROSSLINKED, EMPTY_AREA_8 = CROSSLINKED)
+
+//for modifying jobs
+#define MAP_JOB_CHECK if(SSmapping.config.map_name != JOB_MODIFICATION_MAP_NAME) { return; }
+#define MAP_JOB_CHECK_BASE if(SSmapping.config.map_name != JOB_MODIFICATION_MAP_NAME) { return ..(); }
+#define MAP_REMOVE_JOB(jobpath) /datum/job/jobpath/map_check() { return (SSmapping.config.map_name != JOB_MODIFICATION_MAP_NAME) && ..() }
```

Comment:
See, this:
```MAP_REMOVE_JOB(hos) ... jobpath ...``` will become -> ```/datum/job/jobpath/map_check...``` See what's wrong? No "hos"

Ok, so if you imported it:
```MAP_REMOVE_JOB(hos) ... #jobpath ...``` -> ```/datum/job/ hos /mapcheck()```
What's wrong? Spaces in a path!

```MAP_REMOVE_JOB(hos) ... ##jobpath ...``` -> ```/datum/job/hos/mapcheck()```
Perfect!